tables/th-g0-backward.utb

```diff
@@ -0,0 +1,222 @@
+# liblouis: Thai Current Braille System.
+#
+#-index-name: Thai
+#-display-name: Thai current braille
+#
+#+language:th
+#+region:th-TH
+#+type:literary
+#+system:th-standard
+#+contraction: no
+#+grade: 0
+#+dots: 8
+# Marked as "direction:Backward" by thai braille funds
+# as tests only run backward.
+#+direction:backward
+#
+# Version 2023-06-01
+# Optimize the display of Braille.
+#-copyright: Thai braille fund and FOUNDATION FOR THE BLIND IN THAILAND UNDER THE ROYAL PATRONAGE OF H.M. THE QUEEN.
+#
+#  Copyright (C) 2023 by Thai braille fund and FOUNDATION FOR THE BLIND IN THAILAND UNDER THE ROYAL PATRONAGE OF H.M. THE QUEEN.
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# Created and maintained by: Chatupol Nootatong <jpt@tab.or.th> 
+#	- Dr.Wiraman Niyomphol <wiraman.n@gmail.com>
+#	- Itsawat Banlawanich <peet.itsawat@gmail.com>
```

Comment:
`#-author` and `#-maintainer` tags should be created from this. Typically there can be many authors, and one of them is the maintainer (although there could be multiple maintainers too).

Reply:
So, just to be clear, you mean like this?

```
#-maintainer: Chatupol Nootatong <jpt@tab.or.th> 
#-maintainer: Dr. Wiraman Niyomphol <wiraman.n@gmail.com>
#-maintainer: Itsawat Banlawanich <peet.itsawat@gmail.com>
```

Reply:
#-author: Chatupol Nootatong <jpt@tab.or.th> 
#-author: Dr. Wiraman Niyomphol <wiraman.n@gmail.com>
#-author: Itsawat Banlawanich <peet.itsawat@gmail.com>

Can you write like this? If you can't write like this Write the way you have presented.

Reply:
Can one of you be the maintainer and main contact person?

Reply:
yes I can